frontend/packages/ceph-storage-plugin/src/components/block-pool/dashbaord/compression-details-card.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { compose } from 'redux';
+
+import { getInstantVectorStats } from '@console/internal/components/graphs/utils';
+
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import DetailItem from '@console/shared/src/components/dashboard/details-card/DetailItem';
+import DetailsBody from '@console/shared/src/components/dashboard/details-card/DetailsBody';
+import { usePrometheusQueries } from '@console/shared/src/components/dashboard/utilization-card/prometheus-hook';
+
+import { BlockPoolDashboardContext } from './block-pool-dashboard-context';
+import { getPoolQuery, StorageDashboardQuery } from '../../../queries/ceph-queries';
+
+const parser = compose((val) => val?.[0]?.y, getInstantVectorStats);
+
+export const CompressionDetailsCard: React.FC = () => {
+  const { t } = useTranslation();
+  const { obj } = React.useContext(BlockPoolDashboardContext);
+  const { compressionMode } = obj.spec;
+  const compressionEnabled = compressionMode !== 'none';
+  const { name } = obj.metadata;
+
+  // Metrics
+  const queries = React.useMemo(
+    () => [
+      getPoolQuery([name], StorageDashboardQuery.POOL_COMPRESSION_SAVINGS),
+      getPoolQuery([name], StorageDashboardQuery.POOL_COMPRESSION_ELIGIBILITY),
+      getPoolQuery([name], StorageDashboardQuery.POOL_COMPRESSION_RATIO),
+    ],
+    [name],
+  );
+
+  const [values, loading] = usePrometheusQueries(queries, parser as any);
+  const poolCompressionSavings = (values?.[0] ?? 0) as number;
+  const poolCompressionEligibility = (values?.[1] ?? 0) as number;
+  const poolCompressionRatio = (values?.[2] ?? 0) as number;
```

Comment:
None of the metrics are humanized here
you have to use 
humanizePercentage  for poolCompressionEligibility
humanizeBinaryBytes for  poolCompressionSavings
is poolCompressionRatio is just a number? (no need to do like a:b)?

Reply:
Humanized all the numbers. Yes, Ratio is just a number and not something shown in `x : y` format.

Reply:
ack

Reply:
Rather than defaulting to 0. We should be showing :
1. "not available" : when no values/ empty values
2.  when its actually 0

Reply:
@afreen23 , for your above comment (https://github.com/openshift/console/pull/9507#discussion_r673313723), I have updated the code a bit differently.

Removed the default ZERO value, as we won't enter into the default state if the query is still being loaded (or we hit any query error). Please take a look.